<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="image">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>561</width>
            <height>500</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>18 м &gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="sightSlider">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_sight">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maxLength">
             <number>30</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Стрелы Eastone 23/14</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="5" column="0">
              <widget class="QLabel" name="label_35">
               <property name="text">
                <string>Оперение</string>
               </property>
              </widget>
             </item>
             <item row="3" column="5">
              <widget class="QLabel" name="label_31">
               <property name="text">
                <string>gn</string>
               </property>
              </widget>
             </item>
             <item row="4" column="5">
              <widget class="QLabel" name="label_34">
               <property name="text">
                <string>мм</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_27">
               <property name="text">
                <string>Диаметр древка</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_32">
               <property name="text">
                <string>Вес стрелы</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label_30">
               <property name="text">
                <string>Начальная скорость стрелы</string>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <widget class="QLabel" name="label_29">
               <property name="text">
                <string>fps</string>
               </property>
              </widget>
             </item>
             <item row="2" column="5">
              <widget class="QLabel" name="label_33">
               <property name="text">
                <string>г</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QLineEdit" name="lineEdit_speed_ms"/>
             </item>
             <item row="0" column="5">
              <widget class="QLabel" name="label_28">
               <property name="text">
                <string>м/с</string>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QLineEdit" name="lineEdit_speed_fps"/>
             </item>
             <item row="2" column="4">
              <widget class="QLineEdit" name="lineEdit_weight_g"/>
             </item>
             <item row="3" column="4">
              <widget class="QLineEdit" name="lineEdit_weight_gn"/>
             </item>
             <item row="4" column="4">
              <widget class="QLineEdit" name="lineEdit_diameter"/>
             </item>
             <item row="5" column="4">
              <widget class="QLineEdit" name="lineEdit_feather"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>268</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="6">
         <widget class="QSlider" name="distanceSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="lineEdit_distance">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>450</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>450</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>м</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1132</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>distanceSlider</tabstop>
  <tabstop>lineEdit_distance</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
